standard/standard/use_cases/use_cases.adoc

```diff
@@ -1,18 +1,24 @@
+The GeoPose use cases involve interactions between information systems or between an
+information system and a storage medium. The essential role of a GeoPoses is to
+convey the position and orientation of a real or virtual object. The possibility of
+chained transformational relationships and cross-linkages between chains affords
+representation of complex pose relationships and a way to bring a collection of
+related GeoPoses in a common geographic reference frame.
 
-
-[[test_bookmark]]
-
-The GeoPose use cases involve interactions between information systems or between an information system and a storage medium. The essential role of a GeoPoses is to convey the position and orientation of a real or virtual object. The possibility of chained transformational relationships and cross-linkages between chains affords representation of complex pose relationships and a way to bring a collection of related GeoPoses in a common geographic reference frame.
-
-Each use case is identified by a unique ID, has a brief description, and a list of the relevant <<def_standardization_targets,**Standardization Targets**>>
+Each use case is identified by a unique ID, has a brief description, and a list of
+the relevant <<def_standardization_targets,*Standardization Targets*>>
 
 ==== Augmented and Mixed Reality [AR]
 
-**Description**: Augmented reality (AR) integrates synthetic objects or synthetic representations of real objects with a physical environment. Geospatial AR experiences can use GeoPose to position synthetic objects or their representations in the physical environment. The geospatial connection provides a common reference frame to support integration in AR.
+*Description*: Augmented reality (AR) integrates synthetic objects or synthetic
+representations of real objects with a physical environment. Geospatial AR
+experiences can use GeoPose to position synthetic objects or their representations in
+the physical environment. The geospatial connection provides a common reference frame
+to support integration in AR.
 
 ===== Use Cases:
 
-[%header,cols="2,5,2"] 
+[%unnumbered,%header,cols="2,5,2"]
```

Comment:
Why is this "unnumbered"?

Reply:
@ronaldtse that comes from original source as well.
Maybe they don't want these tables to get numbered?

Reply:
I'm not seeing in the diff how the `unnumbered` part came from the original source. When the original source does not explicitly specify `unnumbered`, let's not add it. Tables are by default numbered in OGC except for very special circumstances.

Reply:
Huh, it turns out that the original markup doesn't include any `unnumbered` attribute, as you say.
I was looking at the PDF file, which shows the tables without any numbering.
Ok, I'm reverting that change.